start-page/release/index.html

```diff
@@ -10,26 +10,37 @@
       </header>
       <div>
         <section>
-          <h2><i class="far fa-life-ring fa-lg" style="color:#0060df"></i> {{ _('<a href="%(support)s">Get Support</a>')|format(support='https://support.mozilla.org/products/thunderbird') }}</h2>
-          <p>{{ _('Need help with something in Thunderbird. We have a <a href="%(support)s">support page</a> where you ask questions and read knowledge base articles to learn how to use Thunderbird.')|format(support='https://support.mozilla.org/products/thunderbird') }}</p>
+          <h2>{{ _('Why we need donations') }}</h2>
+              <p>
+              {{ _('<strong>Thunderbird improvements are no longer paid for by Mozilla.</strong> Fortunately there is an active community keeping it running and developing it further. But to survive long term, the project needs funding.') }}
+              </p>
+              <p>
+              {{ _('Thunderbird is currently transitioning to an independent organization. Being independent, we can shape our own fate, but there is significant infrastructure that must be maintained to deliver the application to our tens of millions of users. For Thunderbird to survive and continue to evolve, we need your support and ask for your donation today. All the money donated will go directly to funding Thunderbird development and infrastructure.') }}
+              </p>
         </section>
+      </div>
+      <div>
         <section>
-          <h2><i class="fas fa-bug fa-lg" style="color:#058b00"></i> {{ _('<a href="%(bug)s">Report a Bug</a>')|format(bug='https://bugzilla.mozilla.org/describecomponents.cgi?product=Thunderbird') }}</h2>
-          <p>{{ _('Think you\'ve found a problem with Thunderbird? <a href="%(bug)s">Create a bug report</a> so that we can make sure it gets solved. Helping us identify and fix bugs makes for a better Thunderbird for all.')|format(bug='https://bugzilla.mozilla.org/describecomponents.cgi?product=Thunderbird') }}</p>
+          <h2><i class="fas fa-heart fa-lg"  style="color:#ff0039"></i> {{ _('<a href="%(donate)s">Donate to Thunderbird</a>')|format(donate='https://donate.mozilla.org/en-US/thunderbird/?presets=100,50,25,15&amount=50&utm_source=start_page_thunderbird&utm_medium=referral&utm_content=header&currency=usd') }}</h2>
+            <p>{{ _('<strong>Thunderbird is completely funded by donations!</strong> Thunderbird is the leading open source, cross-platform and calendaring client, free for business and personal use. We want it to stay secure and become even better. A donation will allow us to hire developers, pay for infrastructure, and continue to improve.') }}</p>
```

Comment:
"Completely" is almost true, but there are minor income streams and those might grow, so I think this needs adjustment as well.

Reply:
On a general note, it's more effectful with the boldings if they are not the first thing after the header.

Reply:
Let me know what you think of the changes I made, used bold in different places.